Index: driver-intel-ixgbe-5.5.2/src/ixgbe_main.c
===================================================================
--- driver-intel-ixgbe-5.5.2/src/ixgbe_main.c
+++ driver-intel-ixgbe-5.5.2/src/ixgbe_main.c	2021-03-01 17:12:34.382123915 +0100
@@ -1826,7 +1826,7 @@
 	 * number of references the driver holds.
 	 */
 	if (unlikely(pagecnt_bias == 1)) {
-		page_ref_add(page, USHRT_MAX);
+		page_ref_add(page, USHRT_MAX - 1);
 		rx_buffer->pagecnt_bias = USHRT_MAX;
 	}
 #else
